RenderSystems/GLES2/src/OgreGLES2RenderSystem.cpp

```diff
@@ -839,10 +839,14 @@ namespace Ogre {
 
     void GLES2RenderSystem::_setSampler(size_t unit, Sampler& sampler)
     {
+        GLenum target = mTextureTypes[unit];
+
+        if (target == GL_TEXTURE_EXTERNAL_OES)
+            return;
+
         if (!mStateCacheManager->activateGLTextureUnit(unit))
             return;
 
-        GLenum target = mTextureTypes[unit];
 
```

Comment:
the early out should probably only be done here so the texture unit still gets activated

Reply:
I tried to return my return, it led to INVALID_ENUM error. Than I added correct parameters according to https://www.khronos.org/registry/OpenGL/extensions/OES/OES_EGL_image_external.txt and google example (latest changes). And I still could no see camera image, although no gl error occurs now